support-frontend/assets/helpers/storage/abandonedBasketCookies.ts

```diff
@@ -59,36 +39,6 @@ function parseAmount(amount: number): number | 'other' {
 	return isNaN(amount) ? 'other' : amount;
 }
 
-export function updateAbandonedBasketCookie(amount: string) {
```

Comment:
This is now unused, should we be setting and updating the value in the one time checkout? If so it was probably missed in the migration to the new one time checkout. @tomrf1 maybe you have some thoughts

Reply:
Yes the abandoned basket cookie is still in use! Does it still work for other checkouts?

Reply:
Yes, it's used in the generic checkout (so RC, S+, Tier 3)